scripts/clean.py

```diff
@@ -0,0 +1,83 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# Azure Killer
+# version 0.1
+# Clean Azure resources automatically
+
+import os
+import json
+import datetime
+
+# Please set parameters
+
+# Subscription ID
+subscription = '0b1f6471-1bf0-4dda-aec3-cb9272f09590'
+
+# Prefix of resource group that will not be deleted
+prefixes = []
+
+# Maximum survival time, in days
+TTL = 3
+
+
+def main():
+    print('Azure Killer, version 0.1')
+    print('Configuration:')
+    print('    Subscription: ' + subscription)
+    print('    Resource group prefix: ' + str(prefixes))
+    print('    Maximum survival time: %d days' % TTL)
+    print()
+    cmd = 'az group list --subscription %s --query [].name'
```

Comment:
I ran this command locally. It gives me `zsh: no matches found: [].name`. Is it some known issue with zsh?

Reply:
I only tested it in Powershell. It's JMESPath syntax